index.bs

```diff
@@ -651,8 +716,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
     
     1. [=set/Append=] |authenticator| to |issuedRequests|.
 
-1. Let |promise| be [=a new Promise=]. Return |promise| and start a timer for |adjustedTimeout| milliseconds. Then execute the
-    following steps [=in parallel=]. The [=task source=] for these [=tasks=] is the [=dom manipulation task source=].
+1. Start a timer for |adjustedTimeout| milliseconds. Then execute the following steps [=in parallel=]. The [=task source=] for
```

Comment:
alrightie -- i wonder whether that rationale ought to be captured somewhere in these spec(s)?